astropy/visualization/interval.py

```diff
@@ -6,39 +6,73 @@
 """
 
 from __future__ import division, print_function
-
```

Comment:
why remove the blank line?

I always thought they were meant to seperate builtins, externals and internal packages